Formula/gptfdisk.rb

```diff
@@ -27,8 +34,26 @@ def install
   end
 
   test do
-    system "hdiutil", "create", "-size", "128k", "test.dmg"
+    system "dd", "if=/dev/zero", "of=test.dmg", "bs=1m", "count=1"
+    output = shell_output("#{bin}/sgdisk -o test.dmg")
+    assert_match "completed successfully", output
+    output = shell_output("#{bin}/sgdisk -p test.dmg")
+    assert_match "GUID", output
```

Comment:
```suggestion
    assert_match "completed successfully", shell_output("#{bin}/sgdisk -o test.dmg")
    assert_match "GUID", shell_output("#{bin}/sgdisk -p test.dmg")
```
This strikes me as more readable, but I don't have too strong an opinion about it.

Reply:
I don't really either, but I was just sticking with the style that this formula already has :shrug: